### DISCUSSION TITLE
Multi-Output GP using simple shift/scale approach and single kernel

### DISCUSSION THREAD

**Discussion by ingmarschuster** [thomaspinder/GPJax#284] [Ideas]
Previously in [JaxRk](https://github.com/exazyme/JaxRK) I implemented a simple multi-output GP model that works as follows.
A single kernel for all output dimensions, let Y1, …, Yd be the data points for each individual output dimension. We center and scale them to standard deviation 1 resulting in Z1, …, Zd.
The predicted mean for dimension j then is computed as Zj inv(kernel.gram(train)) kernel.cross_covariance(train, test) + np.mean(Yj), the convariance for dimension j is computed by scaling the standard GP predictive covarinace by np.std(Yj).

Would you be open to an approach like this? Its rather simplistic but has often served me well and does not need a very fancy implementation.

**Comment by thomaspinder** [comment]
Hey @ingmarschuster. Yes, I'd be very happy to get an approach like this in. 
It'd be wonderful one day to have a fully-fledged API for multi-output GPs but, time... The simplicity of this approach would give us a simple MO GP that we could use to refine an API design.
